src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3409,6 +3412,12 @@ def update_vmss(cmd, resource_group_name, name, license_type=None, no_wait=False
     if user_data is not None:
         vmss.virtual_machine_profile.user_data = b64encode(user_data)
 
+    if v_cpus_available is not None:
+        vmss.virtual_machine_profile.hardware_profile.vm_size_properties.v_cpus_available = v_cpus_available
+
+    if v_cpus_per_core is not None:
+        vmss.virtual_machine_profile.hardware_profile.vm_size_properties.v_cpus_per_core = v_cpus_per_core
```

Comment:
Is it possible that `vmss.virtual_machine_profile.hardware_profile` or `vmss.virtual_machine_profile.hardware_profile.vm_size_properties` will be none? Could you ensure that there is no None pointer exception here?